<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mortar</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/mortar/icons/icon.ico</normaloff>:/mortar/icons/icon.ico</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="2">
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <item row="2" column="1">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Manual azimuth correction</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Manual altitude correction</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QPushButton" name="shutterModeButton">
        <property name="text">
         <string>Enable shutter mode</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Azimuth</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QTimeEdit" name="timeEdit">
        <property name="displayFormat">
         <string>HH:mm:ss</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>GPS time (UTC)</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="startStopButton">
        <property name="text">
         <string>Start Stop mode disabled</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>GPS location</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="labelManAlt">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QDateEdit" name="dateEdit"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="labelAltitude">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="labelInterval">
        <property name="text">
         <string>Start Stop interval [s]: 10</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QSlider" name="startStopIntervSlider">
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>60</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="pauseButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Pause Movement</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="syncButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Sync with Stellarium</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="powerDownButton">
        <property name="text">
         <string>Disable Steppers</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="labelManAzi">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="resAltButton">
        <property name="text">
         <string>Reset Counter</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelAzimuth">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Altitude</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="resAziButton">
        <property name="text">
         <string>Reset Counter</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QLineEdit" name="locationEdit"/>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>GPS date</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>80</height>
         </size>
        </property>
        <property name="title">
         <string>Select tracked object</string>
        </property>
        <widget class="QRadioButton" name="virtTeleRadioButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>131</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Virtual telescope</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="selObjRadioButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>131</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Current object</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="8" column="3">
       <widget class="QPushButton" name="syncGPSButton">
        <property name="text">
         <string>Sync Stellarium with GPS</string>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Tracked object name</string>
        </property>
       </widget>
      </item>
      <item row="9" column="3">
       <widget class="QLabel" name="trackedObjLabel">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="altDownButton">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="altUpButton">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="3">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="aziLeftButton">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="aziRightButton">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>577</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Mortar</string>
    </property>
    <addaction name="actionMotor_Preferences"/>
    <addaction name="actionCorrection_values"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionMotor_Preferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionCorrection_values">
   <property name="text">
    <string>Correction values</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
